src/main/java/com/project/momo/entity/AdjustPoint.java

```diff
@@ -0,0 +1,27 @@
+package com.project.momo.entity;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "ADJUST_POINTS")
+public class AdjustPoint extends BaseEntity{
```

Comment:
```suggestion
public class AdjustPoint extends BaseEntity {
```
찾아라 컨벤션! 👀 
항상 다른사람과의 협업 가독성을 위해서 코드 컨밴션은 지켜주세요.
인텔리 제이 내에서 window:`ctrl + alt + l`, mac:`opt + cmd + l`

Reply:
리뷰 감사합니다!!
미처 생각을 못했었는데 가독성에 영향을 줄 수 있겠네요.
컨벤션 지키며 작성하겠습니다!!.😀